### DISCUSSION TITLE
can not connect remote jupyter Hub

### DISCUSSION THREAD

**Discussion by liudaolunboluo** [microsoft/vscode-jupyter#15876] [Questions and Answers]
<img width="750" alt="image" src="https://github.com/user-attachments/assets/a2da473a-a48c-4cd8-97f6-9e380e5cc04c">

i'm sure My username and password can login jupyter hub

**Comment by DonJayamanne** [comment]
Sorry you are running into this issue.
Please could you enable logging as follows:
* Open settings & go into `Jupyter->Logging`
* Change the value to `trace`
* Reload VS Code,
* Attempt to repro the issue & then copy the output from the `Jupyter` output panel (use the command `Jupyter: Show Output` to get to the logs).
